data/json/recipes/ammo/components.json

```
@@ -421,5 +444,602 @@
+      [ [ "hydraulic_press", 10 ] ],
+      [ [ "forging_standard", 1, "LIST" ] ],
+      [ [ "swage", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ]
+    ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "357mag_casing",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "5 m",
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 3 ] ],
+    "tools": [
+      [ [ "hydraulic_press", 10 ] ],
+      [ [ "forging_standard", 1, "LIST" ] ],
+      [ [ "swage", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ]
+    ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "9x18mm_casing",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": "5 m",
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 3 ] ],
+    "tools": [
+      [ [ "hydraulic_press", 10 ] ],
+      [ [ "forging_standard", 1, "LIST" ] ],
+      [ [ "swage", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ]
+    ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "380_casing",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "5 m",
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 3 ] ],
+    "tools": [
+      [ [ "hydraulic_press", 10 ] ],
+      [ [ "forging_standard", 1, "LIST" ] ],
+      [ [ "swage", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ]
+    ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ] ]
+  },
+  {
+    "result": "300blk_casing",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "5 m",
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_rifle", 3 ] ],
+    "tools": [
+      [ [ "hydraulic_press", 10 ] ],
+      [ [ "forging_standard", 1, "LIST" ] ],
+      [ [ "swage", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ]
+    ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 5 ] ] ]
+  },
+  {
+    "result": "shot_hull",
+    "type": "recipe",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_COMPONENTS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "time": "5 m",
+    "book_learn": [ [ "recipe_bullets", 2 ], [ "mag_shotgun", 3 ] ],
+    "tools": [
+      [ [ "hydraulic_press", 10 ] ],
+      [ [ "forging_standard", 1, "LIST" ] ],
+      [ [ "swage", -1 ], [ "small_repairkit", -1 ], [ "large_repairkit", -1 ] ],
+      [ [ "mold_plastic", -1 ] ]
+    ],
+    "charges": 1,
+    "components": [ [ [ "cartridgebrass", 3 ] ], [ [ "plastic_scrap", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "cartridgebrass", 2 ] ], [ [ "plastic_scrap", 2 ] ] ]
```
2-2 seems the better ratio here, especially given that shotgun shells are mostly plastic

---

Won't work, 1 shell is 8 grams, and plastic scrap is 5 grams. Worth noting that brass is 1 gram per unit and every casing recipe uses the same weight of material the casing weighs, so making it 2-2 would cause you to consume 12 grams of material (10 plastic, 2 brass) to make an 8-gram hull.

---

Oh god, that means there's literally no way to get it to be remotely reasonable brass to plastic ratio atm without messing with *something* here.

Plastic scrap getting changed is obviously out the window for this PR, we evidently don't want to add material wastage to the recipe... only real option in-scope would be fucking with the weight of the shotgun hulls